Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "hades252"
-version = "0.12.0"
+name = "dusk-hades"
```

Comment:
That is fine, but I think we need some additional mandatory entries in `Cargo.toml` to publish in crates.io

Reply:
You're totally right, I always forgot them.